azure/packages/azure-client/src/AzureClient.ts

```diff
@@ -69,10 +69,10 @@ export class AzureClient {
         this.urlResolver = new AzureUrlResolver();
         // The local service implementation differs from the Azure Fluid Relay in blob
         // storage format. Azure Fluid Relay supports whole summary upload. Local currently does not.
-        const enableWholeSummaryUpload = isAzureRemoteConnectionConfig(this.props.connection);
+        const isRemoteConnection = isAzureRemoteConnectionConfig(this.props.connection);
```

Comment:
do we want this change in main as well? how do we think about porting here? release first then port? or main first them port?

Reply:
i just meant the 1.0 release branch that this targets. that partially answers my question. 
in the past i know we preferred main first, and then port, to avoid missing anything in main. i think it would be good for you all to have a perspective about the right way, to help avoid missing ports.

Reply:
Given these 2 were small/immediate PRs I didn't have strong preference what goes in first. Generally, main-first is easier to reason about (simplicity, managing diverging changes etc as Tyler pointed out). @tylerbutler do we call this out anywhere?

Reply:
Typical flow would be main, then cherry-pick as a new PR against release after merge. For most changes that's the safest approach, but this change is really simple so opened both simultaneously.